README.md

```diff
@@ -182,6 +182,8 @@ Free Podcasts and Screencasts:
 
 ### Programming Playgrounds
 
+Write, compile and run your code. Try it out !
```

Comment:
```suggestion
Write, compile and run your code. Try it out!
```

Reply:
Not a suggestion of the text, but a "syntax error"

Reply:
Oh, mb. Ty